tests/Integration/Behaviour/Features/Context/Domain/ProductFeatureContext.php

```diff
@@ -94,6 +97,38 @@ public function addProduct(string $productReference, TableNode $table): void
         }
     }
 
+    /**
+     * @When I pack :quantity :type product :productReference to a pack of :packReference
+     *
+     * @param string $type
+     * @param int $quantity
+     * @param string $productReference
+     * @param string $packReference
+     */
+    public function addProductToPack(string $type, int $quantity, string $productReference, string $packReference)
```

Comment:
I don't understand why you need this type? Maybe there's a particular way to handle combination (will it be a different command?) But if so you can use a different function later Keep this one simple for now without the type